siad/daemon.go

```diff
@@ -91,17 +82,16 @@ func startDaemon() error {
 	// that daemon startup has completed. A gofunc is used with the hope that
 	// srv.Serve() will start running before the value is sent down the
 	// channel.
+	//
+	// TODO: There are better ways to approach this.
```

Comment:
most projects don't test main, which I don't take too much issue with. We don't really test main either.